tests/Unit/PackagePrivate/InMemoryTermIdsAcquirerTest.php

```diff
@@ -91,4 +92,78 @@ private function assertNoDuplicates( $ids ) {
 		);
 	}
 
+	public function testCleanTerms_doesNotReuseIds() {
+		$termIdsAcquirer = new InMemoryTermIdsAcquirer();
+
+		$ids1 = $termIdsAcquirer->acquireTermIds( [
+			'label' => [
+				'en' => 'the label',
+				'de' => 'die Bezeichnung',
+			],
+			'alias' => [
+				'en' => [ 'alias', 'another' ],
+			],
+		] );
+		$ids2 = $termIdsAcquirer->acquireTermIds( [
+			'label' => [ 'en' => 'the label' ],
+			'description' => [ 'en' => 'the description' ],
+		] );
+
+		$termIdsAcquirer->cleanTerms( array_merge( $ids1, $ids2 ) );
```

Comment:
What is the reason for making two calls to acquireTermIds rather than one?

Reply:
I created a PR into this one that changes this test to only one such call: https://github.com/wmde/mediawiki-term-store/pull/17

Reply:
No particular reason, just a vague sense of “let’s make sure the implementation supports multiple `acquireTermIds` calls”.

Reply:
That is not the job of this test method